<launch>

   <node pkg="UKF" name="ukf" type="ukf_estimate" output="screen" >
     <param name="topic_imu" value="/drone1/mavros/imu/data" />
     <param name="topic_mocap" value="/vrpn_client_node/RigidBody1/pose" />
     <param name="topic_thrust" value="/drone1/mavros/vfr_hud" />
     <param name="imu_bias_x" value="101" />
     <param name="imu_bias_y" value="101" />
     <param name="g" value="101" />
     <param name="thrust" value="101" />
   </node>

</launch>
